dot/core/service.go

```diff
@@ -365,8 +367,6 @@ func (s *Service) ProcessBlockAnnounceMessage(msg network.Message) error {
 		seed := rand.New(s1).Uint64()
 		randomID := mrand.New(mrand.NewSource(int64(seed))).Uint64()
 
-		currentHash := s.blockState.LatestHeader().Hash()
```

Comment:
not sure why this doesn't work any more? not a big deal though, as long as we have number or hash it's fine